lib/nerves_hub_web/components/device_page/health_tab.ex

```diff
@@ -378,8 +374,7 @@ defmodule NervesHubWeb.Components.DevicePage.HealthTab do
   end
 
   defp custom_metrics(metrics) do
-    metrics
-    |> Enum.reject(fn {key, _value} ->
+    Enum.reject(metrics, fn {key, _value} ->
```

Comment:
I'd do it, but you don't have to :P
```suggestion
    Enum.reject(metrics, & elem(&1, 1) in @manual_metrics)
```

Reply:
Love it! I would commit your suggestion but you didn't include the two lines below.